pkg/rpcclient/notary/actor.go

```diff
@@ -332,3 +339,21 @@ func (a *Actor) Wait(mainHash, fbHash util.Uint256, vub uint32, err error) (*sta
 	}
 	return a.WaitAny(context.TODO(), vub, mainHash, fbHash)
 }
+
+// WaitSuccess works similar to [Actor.Wait], but checks that the main
+// transaction was accepted and it has a HALT VM state (executed successfully).
+// [state.AppExecResult] is still returned in case you need some additional
+// event or stack checks.
```

Comment:
BTW, currently it's not true. I think it's better to adjust the implementation and return it.

Reply:
I think it's documentation that needs to be amended. Otherwise it's ambiguous, for some errors you get AER, for some don't.